src/js/component/Buttons/Favorite/Favorite.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import ImageFavorite from "../../../../img/icons/favorite/ImageFavorite.png";
+import GoldenFavorite from "../../../../img/icons/favorite/GoldenFavorite.png";
+import "./Favorite.scss";
+
+/**
+ * ! Creates the Favorite Button
+ * * Keikeka (KeiliRosales) - 2022-02-04
+ * @returns React Component
+ */
+
+const Favorite = () => {
+  return (
+    <>
+    <div className="button-favorite">
+       <img 
+       className="favorite-out"
+       src={ImageFavorite} 
+       alt="Image Favorite Button" 
+       height="52"
+       width="52"/>
+      <img 
+       className="favorite-golden"
+       src={GoldenFavorite} 
+       alt="Image Favorite Button Active" 
+       height="52"
+       width="52"/>
+    </div>
```

Comment:
Only one of those will be rendered at any given time. Each one of these should be a different component: one for the empty star and another for the golden one. If we manage to use it as SVG we could use fill it with CSS, but for the moment, we need two different components.

Reply:
Also, whenever is possible, let's try and avoid using inline style properties (e.g: `"height="52" width="auto"`). In this particular case, these buttons are embedded in the Coll, we have a global class for them, in order to control the size of all buttons easily: `className="button-coll"`